internal/mode/static/nginx/conf/nginx.conf

```diff
@@ -1,5 +1,5 @@
 load_module /usr/lib/nginx/modules/ngx_http_js_module.so;
-include /etc/nginx/module-includes/*.conf;
+include /etc/nginx/main-includes/*.conf;
```

Comment:
@bjee19 pointed out to me offline that I need to update this line in the nginx-plus.conf as well. Good catch @bjee19 

Will update